.eslintrc.json

```diff
@@ -16,7 +16,8 @@
   "rules": {
     "indent": [
       "error",
-      2
+      2,
+      { "SwitchCase": 1 }
```

Comment:
what does this eslint change mean?

Reply:
Good question, @ndushay 

Prefer this:

```javascript
switch(foo) {
  case '1':
    whatever()
  default:
    whatever()
}
```

over:

```javascript
switch(foo) {
case '1':
  whatever()
default:
  whatever()
}
```

But I acknowledge this is subject and others may in fact prefer the latter?

Reply:
personally, I prefer the former.  So I'm good with this change.